lasagne/tests/layers/test_pool.py

```diff
@@ -525,13 +546,107 @@ def test_fail_on_mismatching_dimensionality(self):
         except ImportError:
             pytest.skip("cuDNN not available")
         with pytest.raises(ValueError) as exc:
-            MaxPool2DNNLayer((10, 20, 30), 3, 2)
+            MaxPool2DDNNLayer((10, 20, 30), 3, 2)
         assert "Expected 4 input dimensions" in exc.value.args[0]
         with pytest.raises(ValueError) as exc:
-            MaxPool2DNNLayer((10, 20, 30, 40, 50), 3, 2)
+            MaxPool2DDNNLayer((10, 20, 30, 40, 50), 3, 2)
```

Comment:
Oops, good catch. Forgot that we can't rely on Travis for any tests involving the GPU. I'll remember to test everything locally next time.